api/envoy/extensions/tracers/opentelemetry/samplers/v3/dynatrace_sampler.proto

```diff
@@ -49,5 +50,5 @@ message DynatraceSamplerConfig {
   // - ``root_spans_per_minute`` is unset
   // - ``root_spans_per_minute`` is set to 0
   //
-  uint32 root_spans_per_minute = 5;
+  uint32 root_spans_per_minute = 4;
```

Comment:
We may lost the 4 because negligence or special reasons before. But now it's fact and please don't edit it directly.

Reply:
Yeah it was just an unfortunate overlook. 4 is gone now. Oh well.